docs/en/ingest-management/fleet/add-fleet-server-mixed.asciidoc

```diff
@@ -0,0 +1,223 @@
+(fleet-server/fleet-server.crt) and key (fleet-server/fleet-server.key).
+. Extract the zip file and store it in a secure location.
+. You now have all the necessary certificates to install the {fleet-server} and securely connect it to the hosted {es} instance in the cloud.
+
+[discrete]
+[[fleet-server-add-agents]]
+== Add agents
+
+// What is it / why do you need it?
+Now that the policy exists, you can create agents and add them to the policy.
+These agents will act as {fleet-server}s in your deployment.
+{fleet-server} is just another {agent} in a special operating mode.
+
+// How do you set it up?
+To add agents:
+
+. Return to the {fleet-server} policy. If you've navigated away from the page, here's how you can get there:
+.. In {kib} go to **{fleet}**.
+.. Switch to the **Agent policies** tab.
+.. Select the policy you created previously.
+. Click **Add agent** and follow the instructions to add a {fleet-server}.
+.. Choose the policy name for this deployment.
+.. Choose your deployment model. Quick start mode will be less secure.
+Production mode is the fully secured mode where TLS certificates ensure a secure communication between {fleet-server} and {es}.
+.. Add the {fleet-server} host that was identified earlier. Click **Add host**.
+.. A **Service Token** is required so the {fleet-server} can write data to the connected {es} instance.
+Click **Generate service token** and copy the generated token.
+.. Follow the instructions in {kib} to download, extract, and install the agent.
+The installation instructions will differ based on the deployment mode.
+If deploying in production mode, you will need certificates authenticating
+the {fleet-server} to the {es} to complete the installation.
+.. Copy the installation instructions provided in {kib}, which include some of the known deployment parameters.
+You will need to add the relevant certificates generated in the <<fleet-server-generate-certificate,previous section>>:
++
+[source,sh]
+----
+> sudo ./elastic-agent install  \ --url=https://10.128.0.46:8220 \
+  --fleet-server-es=https://<url of hosted Elasticsearch> \
+  --fleet-server-service-token=<generated service token> \
+  --fleet-server-policy=<created Fleet Server policy> \
+  --certificate-authorities=<PATH to CA>/ca/ca.crt \
+  --fleet-server-cert=<PATH to Fleet-Server>/fleet-server/fleet-server.crt \
+  --fleet-server-cert-key=<PATH to Fleet-Server>/fleet-server/fleet-server.key
+----
+
+. Once the {fleet-server} is installed, it will be enrolled into {fleet}
+and the newly created {fleet-server} policy will be applied to it.
+You can see this on the {fleet-server} policy page:
++
+// image
++
+The {fleet-server} agent will also show up on the main {fleet} page as another agent
+whose life-cycle can be managed (like other agents in the deployment):
++
+// image
+
+[discrete]
+[[fleet-server-install-agents]]
+== Install {agents}
+
+// What is it / why do you need it?
```

Comment:
Yah this is confusing to me, too. I'm not sure why we mention control plane. The main thing is that the agent needs to pass the CA cert in order to connect to Fleet Server over TLS. For connecting securely to Elasticsearch, the CA is passed through a different flag or is specified when the Elasticsearch output is defined. (This is described in [secure-connections.html](https://www.elastic.co/guide/en/fleet/current/secure-connections.html)). There's also the possibility (I think) that this certificate might already be installed on the user's system.

Reply:
control plane is agent --> fleet server connection. So it's about securing that connection.